docs/en-us/dev/user_doc/architecture/design.md

```diff
@@ -2,39 +2,6 @@
 
 Before explain the architecture of the scheduling system, let's first get to know the terms commonly used in scheduling system.
```

Comment:
may need to delete this sentence if remove the Glossary section or make a update.

Reply:
Good catch!